src/components/QnA/comps/Alist.jsx

```diff
@@ -1,9 +1,39 @@
 import React from 'react';
+import styled from 'styled-components';
 import SingleAns from './SingleAns.jsx';
 import AListStyle from '../styles/AListStyle.styled';
+import OuterAListStyle from '../styles/OuterAListStyle.styled';
+import Aletter from '../styles/Aletter.styled';
+
+const LoadAnswers = styled.button`
+  display: flex;
```

Comment:
I don't think I understand why this has `display: flex;` and `justify-content: start;`, since it looks like it just contains text. Would `text-align: left;` produce the same result?